meta-lmp-base/conf/distro/include/non-clangable.inc

```diff
@@ -99,6 +99,9 @@ TOOLCHAIN:pn-trusted-firmware-a = "gcc"
 
 # jetson-agx-xavier-devkit
 TOOLCHAIN:pn-cboot-t19x = "gcc"
```

Comment:
as cboot is been dropped we can drop this as well

Reply:
yup, will remove.

Reply:
done